src/app/components/pool/graphs/heatmap/nodes-heatmap.component.spec.ts

```diff
@@ -144,7 +144,7 @@ describe("NodesHeatmapComponent", () => {
             const title = group.select("title");
 
             expect(title).not.toBeFalsy("Should have a rect in bg");
-            expect(title.text()).toContain("2 tasks running on node");
+            expect(title.text()).toContain(`2 tasks (0/1 slots) running on node (node-${i + 1})`);
```

Comment:
These assertions don't look right - tasks should always consume at least 1 slot.